src/processor/operator/persistent/copy_node.cpp

```diff
@@ -179,9 +179,10 @@ void CopyNode::checkNonNullConstraint(NullColumnChunk* nullChunk, offset_t numNo
 }
 
 void CopyNode::finalize(ExecutionContext* context) {
-    auto numNodes = StorageUtils::getStartOffsetOfNodeGroup(sharedState->getCurNodeGroupIdx()) +
```

Comment:
Why do we make this change? The change looks like a bug to me. This could just lead to 0 numNodes in statistics.